lib/wicked/wlan.pm

```diff
@@ -107,7 +112,17 @@ sub prepare_phys {
     assert_script_run('ip netns add ' . $self->netns_name);
     assert_script_run('ip netns list');
     assert_script_run('iw dev');
-    assert_script_run('iw phy ' . $self->ref_phy . ' set netns name ' . $self->netns_name);
+
+    if (is_sle('<15')) {
```

Comment:
```
my $iw_set_phy_name = 'iw phy ' . $self->ref_phy . ' set netns name ' . $self->netns_name;
if (is_sle('<15')) {
        my $output = script_output(sprintf(q(ip netns exec %s perl -MPOSIX -e '$0="netns_%s_dummy_process"; pause' & echo "BACKGROUND_PROCESS:-$!-"), $self->netns_name, $self->netns_name));
        die("Failed to get netns dummy pid") unless ($output =~ m/BACKGROUND_PROCESS:-(\d+)-/);
        $iw_set_phy_name = 'iw phy ' . $self->ref_phy . ' set netns ' . $1;
}
assert_script_run($iw_set_phy_name);  
```

this is exactly the same but it least for me , more clear defining what is going on here

Reply:
ok